conductr/tasks/install-core.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Copy ConductR
+  copy: src=conductr/files/{{ CONDUCTR_PKG }} dest={{ REMOTE_WORK_DIR }}
```

Comment:
@edwardcallahan - introduced `REMOTE_WORK_DIR` - this is used as part of ConductR core install.